styles/components/_layout.scss

```diff
@@ -425,3 +425,50 @@
 .posF {
   position: fixed;
 }
+.about { 
+  .brandSmall {
+    width: 50px;
+    height: 50px;
+  }
+
+  .gemSmall {
+    width: 16px;
+    height: 16px;
+    margin: 1px;
+    background-size: contain;
+  }
+
+  .flexBar {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .flexSpaceBetween {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  
+  #donation-qr-code {
+    width: 200px; 
+    height: 200px;
+    background: black;
+  } 
+
+  #donate-cta {
+    max-width: 80%;
+    margin: auto;
+  }
+
+  .flex0 {
+    flex: 0;
```

Comment:
See comments above for a better way to get the layout this class was used for.

As a general rule, we probably don't ever want an element to have flex-basis of 0%, which is one of the things this shorthand dose.